### PR TITLE
Introduced GetAllTSSs

### PR DESCRIPTION
**Pull request by abdallahbeshi**
## Description

Introduced GetAllTSSs.

## Type of change
- [ ] Bug fix.
- [x] Feature.
- [ ] Consolidation.

## Checklist
- [ ] Is breaking change.
- [ ] Documentation updated.
- [x] Tests included (please specify cases).

### CODE REVIEW

File: src/Germany/Mews.Fiscalizations.Germany.Tests/V2/TssTests.cs [hunk 1]
```diff
@@ -41,6 +41,16 @@ public async Task GetTssSucceeds()
             AssertTss(tss.IsSuccess, tss.SuccessResult.Id);
         }
 
+        [Test]
+        public async Task GetAllTSSsSucceeds()
+        {
+            var client = TestFixture.GetFiskalyClient();
```
Comment (KaliCZ) [inline src/Germany/Mews.Fiscalizations.Germany.Tests/V2/TssTests.cs:47]: Can we create a TSS here and assert it is in the result?
- Create one
- assert Get
- uninitialize

File: src/Germany/Mews.Fiscalizations.Germany/V2/Model/MultipleTss.cs [hunk 2]
```diff
@@ -0,0 +1,14 @@
+﻿using System.Collections.Generic;
+
+namespace Mews.Fiscalizations.Germany.V2.Model
+{
+    public sealed class MultipleTss
```
Comment (KaliCZ) [inline src/Germany/Mews.Fiscalizations.Germany/V2/Model/MultipleTss.cs:5]: We don't need this. Make it return a collection of Tss and this class doesn't need to exist.

File: src/Germany/Mews.Fiscalizations.Germany/V2/FiskalyClient.cs [hunk 3]
```diff
@@ -61,6 +61,15 @@ public Task<ResponseResult<Tss>> GetTssAsync(AccessToken token, Guid tssId)
             );
         }
 
+        public Task<ResponseResult<MultipleTss>> GetAllTSSsAsync(AccessToken token)
+        {
+            return Client.GetResponseAsync<Dto.MultipleTssResponse, MultipleTss>(
```
Comment (KaliCZ) [inline src/Germany/Mews.Fiscalizations.Germany/V2/FiskalyClient.cs:66]:
```suggestion
            return Client.GetResponseAsync<Dto.MultipleTssResponse, IEnumerable<Tss>>(
```

**Review by oleg-sokolansky — APPROVED** [review]

File: src/Germany/Mews.Fiscalizations.Germany.Tests/V2/TssTests.cs [hunk 4]
```diff
@@ -41,10 +35,36 @@ public async Task GetTssSucceeds()
...
+            // In order to disable a TSS, it must be in Uninitialized state.
+            await client.UpdateTssAsync(accessToken, tss.Id, TssState.Uninitialized);
+
+            var newPin = "1234567890";
+            await client.ChangeAdminPinAsync(accessToken, tss.Id, adminPuk, newPin);
+            await client.AdminLoginAsync(accessToken, tss.Id, newPin);
+
+            // Disabling the TSS after creation so we don't exceed the test environment limit.
+            await client.UpdateTssAsync(accessToken, tss.Id, TssState.Disabled);
+        }
```
Comment (KaliCZ) [inline src/Germany/Mews.Fiscalizations.Germany.Tests/V2/TssTests.cs:68]: This can't be named DisableTss. This needs to be 2 different methods

**Review by KaliCZ — APPROVED** [review]

**Pull request merged**